vivarium/core/engine.py

```diff
@@ -694,28 +667,16 @@ def _invoke_process(
 
         Args:
             process: The process.
-            path: The path at which the process resides. This is used to
-                track parallel processes in ``self.parallel``.
             interval: The timestep for which to compute the update.
             states: The simulation state to pass to
                 :py:meth:`vivarium.core.process.Process.next_update`.
 
         Returns:
             The deferred simulation update, for example a
-            :py:class:`vivarium.core.process.ParallelProcess` or an
-            :py:class:`InvokeProcess` object.
+            :py:class:`vivarium.core.process.ParallelProcess`.
         """
-        if process.parallel:
-            # add parallel process if it doesn't exist
-            if path not in self.parallel:
-                self.parallel[path] = ParallelProcess(
-                    process, bool(self.profiler))
-            # trigger the computation of the parallel process
-            self.parallel[path].update(interval, states)
-
-            return self.parallel[path]
-        # if not parallel, perform a normal invocation
-        return self.invoke(process, interval, states)
+        process.send_command('next_update', (interval, states))
```

Comment:
This is pretty nice, `send_command` becomes transparent to the `Engine` this way.